elastalert/alerts.py

```diff
@@ -624,3 +624,43 @@ def get_info(self):
         return {'type': 'pagerduty',
                 'pagerduty_service_key': self.pagerduty_service_key,
                 'pagerduty_client_name': self.pagerduty_client_name}
+
+
+class VictorOpsAlerter(Alerter):
+    """ Creates a VictorOps Incident for each alert """
+    required_options = frozenset(['victorops_api_key', 'victorops_routing_key'])
+
+    def __init__(self, rule):
+        super(VictorOpsAlerter, self).__init__(rule)
+        self.victorops_api_key = self.rule['victorops_api_key']
+        self.victorops_routing_key = self.rule['victorops_routing_key']
+        self.victorops_entity_id = self.rule['victorops_entity_id']
```

Comment:
If this is optional, it should be ``self.rule.get('victorops_entity_id')``.

I don't see it used anywhere right now.